src/documents/integrationWorkFlow.md

```diff
@@ -0,0 +1,28 @@
+---
+title: The Work Flow
+layout: default
+category: Overview
+---
+
+The process of saving preferences to the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) is as follows:
+
+1. The First Discovery Tool sends a POST request to the First Discovery Server @ `/user?view=firstDiscovery`. The body of the request is a JSON string of user selected preferences, for example:
+    ```javascript
+    {
+        fluid_prefs_contrast: "default"
+        fluid_prefs_textSize: 1
+        gpii_firstDiscovery_captions: true
+        gpii_firstDiscovery_language: "en-US"
+        gpii_firstDiscovery_onScreenKeyboard: true
+        gpii_firstDiscovery_showSounds: false
+        gpii_firstDiscovery_speak: false
+        gpii_firstDiscovery_speechRate: 1
+        gpii_firstDiscovery_stickyKeys: false
+    }
+    ```
+
+2. The First Discovery Server parses out user selected preferences, follows [OAuth2 Client Credentials Grant API](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Client_Credentials_Grant) provided by the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) to save preferences to the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md);
```

Comment:
I think it would be ok if only the first occurrence of "GPII Preferences Server" on the page was a link.